### DISCUSSION TITLE
Feature request: Publish in npmjs

### DISCUSSION THREAD

**Discussion by yhojann-cl** [FortAwesome/Font-Awesome#20911] [Feature Requests]
### Is your feature request related to a problem?

You cannot use the latest version in Angular or ReactJS projects through NPM packages such as Bootstrap or Fonts Awesome v4.

### Feature description

Please make an official release for the npm package manager and avoid manual embedding of assets for client side applications and hybrid apps.

### Feature request checklist

- [X] The title starts with "Feature request: " and is followed by the requested feature description
- [X] This is a single feature
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by tagliala** [comment]
Hi!

Could you please clarify this request?
